test/test_jit_cuda_fuser.py

```diff
@@ -4971,6 +4971,27 @@ def t(x, y, w0, w1):
         t_jit = torch.jit.script(t)
         self._run_helper(t_jit, t, x0, x1, w0, w1, check_stride=True)
 
+    # Note that curently TS embeds constant tensor in the graph
+    # I believe this is giving me memory leak in CI
+    @unittest.skipIf(os.environ.get('PYTORCH_NO_CUDA_MEMORY_CACHING') is None,
```

Comment:
I have to disable caching allocator, since it's screaming about leaking memory.

I suspect this is coming from embedded constant tensor. I asked the question in pytorch slack. Once someone confirms that, I'll open an issue to track this.